jdbc-perf-logger-spring-boot-starter-test/src/test/java/ch/sla/jdbcperflogger/spring/test/jpa/MyApplicationTest.java

```diff
@@ -43,4 +50,15 @@ public void testDefaultSettings() throws Exception {
         final StatementLog log = (StatementLog) logRecorder.lastLogMessage(2);
         assertTrue(log.getRawSql().contains(SELECT_COUNT_FROM_PERSON));
     }
+
+    @Test
+    public void testWithComments(){
+        String comments = "first param ?, second param in comment ?";
+        //should use named param
+        Query query = entityManager.createQuery("SELECT p from Person p where p.firstName =  ? ",Person.class).setParameter(1,"David").setHint("org.hibernate.comment", comments);
+        query.getResultList();
+        final StatementLog log = (StatementLog) logRecorder.lastLogMessage(2);
+        System.out.println(log.getFilledSql());
+        assertTrue(log.getRawSql().contains(comments));
```

Comment:
I just check with a real hibernate comment, if someone use the hints from JPA Api to set hibernates comments, but can be removed, since we tested it as unit tests

`
String comments = "first param ?, second param in comment ?";
        //should use named param Query 
query = entityManager.createQuery("SELECT p from Person p where p.firstName =  ? ",Person.class)
                .setParameter(1,"David")
                .setHint("org.hibernate.comment", comments);`
> [           main] org.hibernate.SQL                        : /* first param ?, second param in comment ? */ select person0_.id as id1_0_, person0_.first_name as first_na2_0_, person0_.last_name as last_nam3_0_ from person person0_ where person0_.first_name=?

Added 
`assertTrue(!log.getFilledSql().contains(comments));`